modules/viewing-metrics-for-the-single-model-serving-platform.adoc

```diff
@@ -34,7 +34,11 @@ ifdef::upstream,self-managed[]
 ----
 sum(increase(vllm:request_success_total{namespace='${namespace}',model_name='${model_name}'}[${rate_interval}]))
 ----
-
++
+[NOTE]
+--
+To generate certain vLLM runtime metrics, you must first send an inference request. Idle servers do not generate runtime metrics.
```

Comment:
I'm unsure what we mean by _idle servers_ here - need to elaborate?

Reply:
```suggestion
To generate certain vLLM runtime metrics, you must first make an inference request to a deployed model. Deployed servers that are not serving any inference requests do not generate runtime metrics.
```
@syaseen-rh @vaibhavjainwiz  Does this look good?

Reply:
@eturner24, suggesting slight edit: 

_Certain vLLM latency or throughput metrics such as`vllm:time_to_first_token_seconds_sum` and `vllm:e2e_request_latency_seconds_count` are available only after an inference request is processed by a deployed model. To generate and view these metrics, you must first make an inference request to the model._

If we do not want to be specific about the type of metrics, then:

_Certain vLLM metrics such as`vllm:time_to_first_token_seconds_sum` and `vllm:e2e_request_latency_seconds_count` are available only after an inference request is processed by a deployed model. To generate and view these metrics, you must first make an inference request to the model._

Reply:
```suggestion
Certain vLLM metrics are available only after an inference request is processed by a deployed model. To generate and view these metrics, you must first make an inference request to the model.
```
Thanks @syaseen-rh ! I like your suggestions! I'm going to go with the version that doesn't get into the specific metrics to avoid doc bloat. Ran this by @vaibhavjainwiz over Slack and he approved as well.